frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/RenameBackgroundable.java

```diff
@@ -26,8 +26,8 @@
 import com.virtuslab.qual.async.ContinuesInBackground;
 import com.virtuslab.qual.guieffect.UIThreadUnsafe;
 
-@ExtensionMethod(GitVfsUtils.class)
-public class RenameBackgroundable extends Task.Backgroundable {
+@ExtensionMethod({GitVfsUtils.class, Objects.class})
+public class RenameBackgroundable extends SideEffectingBackgroundable {
 
   protected final Project project;
```

Comment:
I guess this field can be dropped at this level now, right?

- [ ] pls review all the derived classes and `project`/`myProject`/etc.

Reply:
Already done in #1568